File: APP/core/Entities/ordenes/ordenesEntity.py
from pydantic import BaseModel
from typing import Optional

class OrdenesEntity(BaseModel):
    id:Optional[str]
    total:Optional[float]
    sede:str
    fechaPedido:Optional[str]
    fechapago:Optional[str]
    status:str
    idCliente:int
    tipoPago:Optional[str]
    idPago:Optional[str]
    
class OrdenesDetalladasEntity(BaseModel):
   idOrden:str
   cliente:str
   total:float
   fecha:str
   hora:str
   status:str                                       
    
    

File: Front-end/administraccion/administracion/src/core/Entities/ordenes/ordenesEntity.ts
export interface OrdenesEntity {
    id: string;
    total: number;
    sede: string;
    fechaPedido: string;
    fechaPago: string;
    status: string;
    idCliente: number;
    tipoPago: string;
    idPago: string;
  };

export interface OrdenesDetalladasEntity{
  idOrden: string;
  cliente: string;
  total:   number;
  fecha:   Date;
  hora:    string;
  status:  string;

}
export interface ConsumoDetalladoEntity {
  nombre:   string;
  cantidad: number;
  precio:   number;
  total:    number;
}
